src/framework/theme/components/chat/chat-message.component.ts

```diff
@@ -193,15 +202,34 @@ export class NbChatMessageComponent {
    */
   @Input() type: string;
 
+  @Input() customMessageData: any;
+
+  private readonly defaultMessageTypes: string[] = ['text', 'file', 'map', 'quote', undefined];
```

Comment:
Forgot to mention, Nebular uses the `protected` access modifier instead of `private` to facilitate components modification through inheritance. Please, update